### PR TITLE
Connect api to bookstore

### PR DESCRIPTION
**Pull request by danielmatama**
I connected my app to an api

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,276 @@
...
+**[Book Store]** 
+  This app is a must have instrument for book lovers. It is a webpage that allows you to add your favourite books and authors and save them.
+  It also allows you to remove any books you may have already read or wish to replace.
+  It will accept all books and authors without fail. Built with HTML, CSS, and JS.
+
+## 🛠 Built With <a name="built-with">HTML_CSS_JS_React_Redux</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
```
Comment (amejid) [inline README.md]: - [ ] Kindly remove every sentence that starts with a `>` as they are meant to be a guide on what to do there. They should be removed after you add the necessary content. 😎️

**Examples**

Screenshot from 2023-03-01 11-52-28
Screenshot from 2023-03-01 11-52-36

File: README.md [hunk 2]
```diff
@@ -1 +1,276 @@
...
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (amejid) [inline README.md]: - [ ] The FAQ section is optional. It would be great if you put content in it or remove it completely. 🧐️

File: README.md [hunk 3]
```diff
@@ -1 +1,276 @@
...
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (amejid) [inline README.md]: - [ ] Kindly remove this part from the README as it is a guide on how to add a license file. 🤝️

**Review by amejid — COMMENTED** [review]
Hi @danielmatama, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional README.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by prolajumokeoni — APPROVED** [review]
Hi @danielmatama 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
  -  Used the Bookstore API to add and remove books 👍  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**